build-system/eslint-rules/unused-private-field.js

```diff
@@ -76,7 +76,7 @@ module.exports = function(context) {
         }
 
         context.report(node, [
-          `Unused private "${name}".`,
+          `Unused private "${name}".`.padEnd(80), // Padding for alignment
```

Comment:
Hm?

Reply:
The `stylish` formatter of `eslint` uses the total length of the message.

https://github.com/eslint/eslint/blob/51132d6a0729b8e54326e9b8f5c8c1f487c8bd31/lib/formatters/stylish.js#L72-L77